modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -174,16 +167,12 @@ class Configuration(object):
 
     @property
     def debug(self):
-        """Debug status
-
-        :param value: The debug status, True or False.
-        :type: bool
-        """
```

Comment:
Why are we removing this comment?
It provides useful info about the type of this property

Reply:
This is a getter not setter so the signature is wrong.

Reply:
Please add these docstrings back in.
When one uses help to find out how to use the methods of a class, it only gives back the docstring in the getter per:
https://stackoverflow.com/questions/16025462/what-is-the-right-way-to-put-a-docstring-on-python-property
Also having the getter describe the type returned by the getter his helpful to users